src/components/TransactionHistoryRows/utils.ts

```diff
@@ -6,34 +6,39 @@ export enum AssetTypes {
   Fee = 'fee'
 }
 
+const fallbackValues = {
+  symbol: ' N/A',
+  precision: 18
+}
```

Comment:
Change to `fallbackValue` - it's a singular value with multiple properties.

Reply:
How about just `fallback`?

Reply:
That works, too!